src/app/cli/src/services/config/models.rs

```diff
@@ -75,6 +76,9 @@ pub struct CLIConfig {
     /// Uploads configuration
     #[merge(strategy = merge_recursive)]
     pub uploads: Option<UploadsConfig>,
+
+    /// Did secret key encryption configuration
+    pub did_encryption: Option<DidSecretEncryptionConfig>,
```

Comment:
I'd like us to have a more hierarchical configuration, grouping options into a few big sections.

In the node config we have `auth` section, where I think DID secrets encryption would fit. Perhaps let's add such a section into CLI and nest this configuration under it?